.github/workflows/build-posix.yml

```diff
@@ -0,0 +1,23 @@
+name: Build (POSIX)
+
+on: [push, pull_request]
+
+jobs:
+  build-posix:
+    runs-on: ubuntu-latest
```

Comment:
POSIX samples can't be run on `ubuntu-latest`, where the thrift package doesn't include the patch with features required. Switch to `ubuntu-22.04` should get it pass.

Reply:
in that case, we may as well specify `ubuntu-20.04` since the `ubuntu-latest` will likely be changed to `ubuntu-22.04` soon.

Reply:
> POSIX samples can't be run on `ubuntu-latest`, where the thrift package doesn't include the patch with features required. Switch to `ubuntu-22.04` should get it pass.

Oh weird - thanks for pointing that out. It's almost counterintuitive that the patch wouldn't be applied for `ubuntu-latest`. Maybe I'll make a note of that in yaml